### PR TITLE
Tests for transformation service integration in java feature server

### PR DESCRIPTION
**Pull request by pyalex**
Signed-off-by: pyalex <moskalenko.alexey@gmail.com>

**What this PR does / why we need it**:

To keep the transformation functionality in Java FS alive.

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: java/serving/src/main/java/feast/serving/service/OnlineTransformationService.java [hunk 1]
```diff
@@ -77,17 +72,18 @@ public OnlineTransformationService(
   @Override
   public TransformFeaturesResponse transformFeatures(
       TransformFeaturesRequest transformFeaturesRequest) {
+    if (this.stub == null) {
+      throw new RuntimeException(
+          "Transformation service endpoint must be configured to " + "enable this functionality.");
```
Comment (woop) [inline java/serving/src/main/java/feast/serving/service/OnlineTransformationService.java:77]: Do we need this ` + `?

Comment (pyalex) [reply java/serving/src/main/java/feast/serving/service/OnlineTransformationService.java]: @woop this is fixed

**Review by woop — APPROVED** [review]

**Pull request merged**